system-image-upgrader

```diff
@@ -324,7 +324,7 @@ do
                 data)
                     for entry in /data/* /data/.writable_image /data/.factory_wipe; do
                         if [ "$USE_SYSTEM_PARTITION" -eq 0 ];then
-                            if [ "$entry" == "/data/rootfs.img" ]; then
+                            if [[ ( "$entry" == "/data/rootfs.img" ) || ( "$entry" == "/data/system.img" ) ]]; then
```

Comment:
Double brackets are bash extension and may not be available in recovery shell. Is it required in this case?

Reply:
Seems its ok with double brackets, for some reason busybox supports bash conditionals ;)